src/lib/mongoBackend/mongoSubCache.cpp

```diff
@@ -200,7 +200,7 @@ int mongoSubCacheItemInsert(const char* tenant, const orion::BSONObj& sub)
 
     if (!entity.hasField(CSUB_ENTITY_ID))
     {
-      LM_W(("Runtime Error (got a subscription without id)"));
+      LM_E(("Runtime Error (got a subscription without id)"));
```

Comment:
Subscription without id shouldn't occur at CB. Worth to be ERROR instead of WARN.

NTC (informative)